examples/music-licensing-challenge/src/app/graphql/mutations.py

```
@@ -0,0 +1,28 @@
+from typing import Optional
+
+from sqlalchemy.orm import Session
+from strawberry import ID, mutation, type
```

Ruff (0.15.13)

[error] Import `type` is shadowing a Python builtin

(A004)